lib/routes/theblockbeats/index.ts

```diff
@@ -1,76 +1,35 @@
 import { Route } from '@/types';
-import cache from '@/utils/cache';
-import got from '@/utils/got';
-import { load } from 'cheerio';
 import { parseDate } from '@/utils/parse-date';
-
-const domain = 'theblockbeats.info';
-const rootUrl = `https://www.${domain}`;
-const apiBase = `https://api.${domain}`;
-
-const channelMap = {
-    newsflash: {
-        title: '快讯',
-        link: `${rootUrl}/newsflash`,
-        api: `${apiBase}/v5/newsflash/select`,
-    },
-    article: {
-        title: '文章',
-        link: `${rootUrl}/article`,
-        api: `${apiBase}/v5/Information/newsall`,
```

Comment:
The api of article is `/v2/article/list`

Reply:
Thanks